### PR TITLE
Expose the version of each adapter

### LINKED ISSUE #36: fury-cli `--version` should emit the adapter versions
Getting the version of fury should emit the versions of the adapters that it will use, so it is clear which parser version is used.

```
$ fury --version
fury 0.8.4
fury-adapter-apib-parser ...
fury-swagger ...
etc
```

### PR DESCRIPTION
**Pull request by kylef**
Closes #36

### CODE REVIEW

File: packages/fury-cli/lib/fury.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env node
 
+/* eslint-disable no-console */
```
Comment (kylef) [inline packages/fury-cli/lib/fury.js]: I didn't add any extra console but there are other `console.log` bringing ESLint to a warning state.

File: packages/fury-cli/lib/fury.js [hunk 2]
```diff
@@ -175,14 +184,27 @@ class FuryCLI {
...
+}
 
 if (require.main === module) {
   let input;
   let output;
 
   commander
     .description(pkg.description)
-    .version(pkg.version)
+    .version(getVersion())
```
Comment (kylef) [inline packages/fury-cli/lib/fury.js]: Ideally, I'd have done something like `.option('--version')` and later did a check for `commander.version` and then computed all the versions. However that isn't possible, the `version` option name is handled in a special way inside Commander and thus if you create a version option you cannot use it later on.

**Comment by erajkovic** [comment]
Not sure how to review / test code change yet.

File: packages/fury-cli/lib/fury.js [hunk 3]
```diff
@@ -175,14 +184,27 @@ class FuryCLI {
   }
 }
 
+function getPackage(name) {
+  // eslint-disable-next-line import/no-dynamic-require, global-require
+  return require(`${name}/package.json`);
```
[the remark this answers is not in the document]
Comment (kylef) [reply packages/fury-cli/lib/fury.js]: I'll change this, the reason at the time is that I wanted it to reflect the `--adapter` option of the parser (which allows users to inject their own adapters).

Unfortunately due to https://github.com/apiaryio/api-elements.js/pull/45#discussion_r241591192 it isn't possible.

[the remark this answers is not in the document]
Comment (honzajavorek) [reply packages/fury-cli/lib/fury.js]: 👍

**Pull request merged**